src/control/ros/RosTrajectoryExecutor.cpp

```diff
@@ -234,7 +234,11 @@ void RosTrajectoryExecutor::step(
 //==============================================================================
 void RosTrajectoryExecutor::abort()
 {
-  // TODO: cancel the actionlib goal (once there is support in ReWD controller)
+  std::lock_guard<std::mutex> lock(mMutex);
+  DART_UNUSED(lock); // Suppress unused variable warning.
+
+  if (mInProgress)
+    mGoalHandle.cancel();
```

Comment:
Would it be useful to print a warning something like "Attempting to abort trajectory execution but no trajectory in progress."?

Reply:
That sounds like a good idea. We should do this to the other `TrajectoryExecutor::abort` implementations as well.